pkg/app/app.go

```diff
@@ -500,8 +500,9 @@ func (a *App) Test(c TestConfigProvider) error {
 func (a *App) PrintState(c StateConfigProvider) error {
 	return a.ForEachState(func(run *Run) (_ bool, errs []error) {
 		err := run.withPreparedCharts("build", state.ChartPrepareOptions{
-			SkipRepos: true,
-			SkipDeps:  true,
+			ForceDownload: false,
```

Comment:
@itscaro ForceDownload default is false.

Reply:
exactly, my mind was away :-), need to find a way to not fetch chart in build then